doc/redirects.csv

```diff
@@ -5,6 +5,9 @@
 whatsnew,whatsnew/index
 release,whatsnew/index
 
+# web
+getting_started,docs/getting_started/index
```

Comment:
Ah, hmm, now realizing this probably won't work for pages outside of `/docs/`. Any maintainers have thoughts about how to handle?

Reply:
Thank you @Afeld !!

Reply:
cc @jorisvandenbossche

Reply:
Thank  you @jreback

Reply:
Indeed, this file is specifically for sphinx page (so under `/docs` in the url), and not for the website. So we will have to do something custom there if we want a redirect (I don't think there is a similar mechanism already). 

In the end, for those redirects listed in this file, we create a small html page with the actual redirection. See 
https://github.com/pandas-dev/pandas/blob/097ff0c34179ab55d08a6ab0757762beea4d8785/doc/make.py#L211-L220

So one option would be to include a single hardcoded html file for this specific redirect, or otherwise add something similar to https://github.com/pandas-dev/pandas/blob/master/web/pandas_web.py which generates the static web pages

Reply:
Yes, the change here in `redirects.csv` should be reverted

Reply:
Okay... Thank you @jorisvandenbossche .. 😄 :

Reply:
After removing the change in this file, let's add `getting_started.html` with [a `<meta>` redirect](https://www.w3.org/TR/WCAG20-TECHS/H76.html#H76-examples) to `/docs/getting_started/index.html`.